qa/tasks/radosgw_admin_rest.py

```diff
@@ -719,3 +722,11 @@ def task(ctx, config):
     (ret, out) = rgwadmin_rest(admin_conn, ['user', 'info'], {'uid' :  user1})
     assert ret == 404
 
+    # TESTCASE 'info' 'info' 'show' 'display info' 'succeeds'
+    (ret, out) = rgwadmin_rest(admin_conn, ['info', ''], {'uid' : user1})
```

Comment:
`{'uid' : user1}` was a parameter passed to the `user` apis above, it isn't necessary for this `info` api request

Reply:
ok, thanks for noticing